requirements.txt

```diff
@@ -12,3 +12,4 @@ progressbar2
 nltk
 numpy
 requests
+psycopg2==2.6.1
```

Comment:
Psycopg is required to run PostgreSQL database for this app. Kiran forgot to add this one to requirements.txt.